server/office.json

```diff
@@ -89,5 +93,10 @@
       "groupId": "lord",
       "icon": "https://cdn.pixabay.com/photo/2015/11/03/09/03/meeting-1019995_960_720.jpg"
     }
-  ]
+  ],
+  "clientConfig": {
```

Comment:
@klassm @endoval Could it be that this part of the office.json is read, but never used?

Reply:
Yes I guess that changed back to env variables.